### DISCUSSION TITLE
Error 500, 403s, and Invalid JSON?

### DISCUSSION THREAD

**Discussion by aanomm** [swellstores/community#319] [Help]
- i have a demo store setup i'm testing Swell with, and while all was well now nothing's well with my Swell
- i'm getting frontend "Error 500" messages and the console shows 403 errors plus `SyntaxError` and more
- here's my demo store: https://www.subb.coffee/products/1
- note: this seems to have started after i added a custom Option field (adding the field seemed to break things so I deleted the field (or Reset)
- 
![image](https://github.com/swellstores/community/assets/1171169/add3bf6a-2d58-4a87-acfe-be5239178d21)

**Comment by steven-k1** [comment]
@fucha-co To clarify, once you deleted the custom option field you mentioned, did the errors still persist?
